code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -325,7 +325,7 @@
 
 /obj/machinery/porta_turret/take_damage(damage, damage_type = BRUTE, damage_flag = 0, sound_effect = 1)
 	. = ..()
-	if(.) //damage received

```

Comment:
This should be using the following proc and `integrity_failure` should be set.

https://github.com/tgstation/tgstation/blob/34e523fd1f5d6b6593539b065a619507716dd666/code/game/objects/obj_defense.dm#L254-L255